scripts/scraper-ng/test/framework/utils.js

```diff
@@ -66,7 +66,57 @@ function recipePath(recipeName) {
   return path.join(recipesDir, recipeName + ".yaml");
 }
 
+/**
+ * Given an ingredient, an ingredient name and a tag name, check that:
+ * * the ingredient exists
+ * * the ingredient name matches the name given
+ * * the ingredient's position node is an element and its tag matches the given tag.
+ *
+ * @param {String} ingredient - an ingredient including a name and a position node
```

Comment:
Isn't `ingredient` supposed to be an object? This comes up throughout, so I wasn't completely sure